rma_repair/models/rma_order.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import api, fields, models
+
+
+class RmaOrder(models.Model):
+    _inherit = "rma.order"
+
+    @api.multi
+    def _compute_repair_count(self):
+        for rma in self:
+            repairs = rma.mapped('rma_line_ids.repair_ids')
+            rma.rma_count = len(repairs)
```

Comment:
```rma_count```? Should not be ```repair_count```?

Reply:
I just created a PR for that https://github.com/grindtildeath/stock-rma/pull/1/